### ISSUE TITLE
(feature)about 2girls

### ISSUE THREAD

**Issue by Chengym2023** [KohakuBlueleaf/z-tipo-extension#60]
1. `1girl, elijah celestial, east asian mythology,Jubilant, 1girl, beau \(fuze\), anger face expression`,
and Can generate character tags and mutual actions,like side-by-side,or other popular actions in danbooru?i think llm genarate action is better than human.

**Comment by KohakuBlueleaf** [comment]
no, TIPO is designed for "RAW danbooru tags", it won't support multiple same tag for "different subject" you should always manage this kind of things by you
especially there is no dataset designed for it, which means this is not intended in TIPO's concept.

For more information on what TIPO is designed for, please check our paper:
https://arxiv.org/abs/2411.08127
